<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpViewersPage</class>
 <widget class="QWidget" name="HelpViewersPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>634</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure help viewers&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line17">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Help Viewer</string>
     </property>
     <layout class="QGridLayout">
      <item row="1" column="0" colspan="4">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLineEdit" name="customViewerEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Enter the custom viewer to be used</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="customViewerSelectionButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Press to select the custom viewer via a file selection dialog</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="customViewerButton">
        <property name="toolTip">
         <string>Select to use a custom viewer</string>
        </property>
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="qtAssistantButton">
        <property name="toolTip">
         <string>Select to use Qt Assistant</string>
        </property>
        <property name="text">
         <string>Qt Assistant</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="helpBrowserButton">
        <property name="toolTip">
         <string>Select to use the Eric Web Browser</string>
        </property>
        <property name="text">
         <string>Eric Web Browser</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="webBrowserButton">
        <property name="toolTip">
         <string>Select to use the configured web browser of the system</string>
        </property>
        <property name="text">
         <string>System Web Browser</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>479</width>
       <height>121</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>helpBrowserButton</tabstop>
  <tabstop>qtAssistantButton</tabstop>
  <tabstop>webBrowserButton</tabstop>
  <tabstop>customViewerButton</tabstop>
  <tabstop>customViewerEdit</tabstop>
  <tabstop>customViewerSelectionButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>customViewerButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>customViewerEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
